docs/tutorials/image_prediction/dataset.md

```diff
@@ -0,0 +1,112 @@
+# Image Prediction - Properly load any type of data as ImagePredictor Dataset
+:label:`sec_imgdataset`
+
+Preparing the dataset for ImagePredictor is not difficult at all, however, we'd like to introduce the
+recommended ways to initialize the dataset so you will have smoother experience using `autogluon.vision.ImagePredictor`.
+
+There are generally three ways to load a dataset for ImagePredictor:
+- Load a csv file or construct your own `DataFrame` with `image` and `label` columns
```

Comment:
add newlines between each of these bullets, otherwise it all appears on the same line.